tsconfig.json

```diff
@@ -4,7 +4,9 @@
         "strictFunctionTypes": false,
         "strictBindCallApply": false,
         "noImplicitThis": false,
-        "target": "ES5",
+        "target": "ES6",
+        "module": "ES6",
```

Comment:
The value of `module` does not require overriding here:
https://github.com/mightyiam/tsconfigs/blob/7cfbd4100e10f6c6bf0bd605a3ace616036b3e96/browser-module.json#L5